update packages for 7.5.0

stormi opened the pull request.

stormi committed:
Update xenserver-release to 7.5.0

Mostly changes to avoid listing 7.4 hotfixes.

stormi committed:
Further changes for 7.5

- use updated branding package
- fix source directory name in %setup

olivierlambert commented on SPECS/xenserver-release.spec:
```diff
@@ -33,10 +33,12 @@ Obsoletes:      control-xscontainer = 1.0-1
 Obsoletes:      update-XS73E001 control-XS73E001
 Obsoletes:      update-XS73E002 control-XS73E002
+Obsoletes:      update-XS73E003 control-XS73E003
+Obsoletes:      update-XS74 control-XS74
```
FYI, this is related to XS patches using their "pack" system. Which is replaced by just a yum repo on our side.

stormi replied:
But it's still useful to ease upgrade from XenServer 7.4,with the patches applied, I suppose. That will make sure the hotfix packages are removed upon upgrade.

stormi committed:
Add missing rpm macros to xenserver-release.spec

XenServer build process probably specifies them as parameters to
rpmbuild. We'll have them inside the spec file for now.

stormi committed:
Switch xcp-ng repo from 7.4 to 7.x

stormi committed:
Change source archive name to xcp-ng-release-xxx

stormi committed:
Run yum clean all in %post after modifying the yum repository file

- It might prevent obsolete metadata or other cached data from being
used after the change.
- Also add a test so that the %post script is run only during RPM
upgrades.

stormi committed:
Add spec file and small source files for XCP-ng Center

Package name remains xencenter-xxx.rpm for now.

stormi committed:
Add specfile for xcp-ng-pv-tools

stormi committed:
Fix bug due to hardcoded XenServer string in xcp-python-libs

This would render some commands inoperant, e.g.:
`/opt/xensource/libexec/xen-cmdline --set-xen dom0_mem=<nn>M,max:<nn>M`

stormi committed:
Add spec file for xcp-ng-updater
